packages/react-examples/src/react/DetailsList/DetailsList.Basic.Example.tsx

```diff
@@ -1,118 +1,168 @@
 import * as React from 'react';
-import { Announced } from '@fluentui/react/lib/Announced';
-import { TextField, ITextFieldStyles } from '@fluentui/react/lib/TextField';
-import { DetailsList, DetailsListLayoutMode, Selection, IColumn } from '@fluentui/react/lib/DetailsList';
-import { MarqueeSelection } from '@fluentui/react/lib/MarqueeSelection';
-import { mergeStyles } from '@fluentui/react/lib/Styling';
-import { Text } from '@fluentui/react/lib/Text';
 
-const exampleChildClass = mergeStyles({
-  display: 'block',
-  marginBottom: '10px',
-});
-
-const textFieldStyles: Partial<ITextFieldStyles> = { root: { maxWidth: '300px' } };
-
-export interface IDetailsListBasicExampleItem {
-  key: number;
-  name: string;
-  value: number;
-}
-
-export interface IDetailsListBasicExampleState {
-  items: IDetailsListBasicExampleItem[];
-  selectionDetails: string;
-}
+import {
```

Comment:
I've only updated this file to make it simple to test the changes to List. There will be no changes made here in an actual PR, though there will likely be a new story demonstrating this rendering behavior.